tools/configure/configure.mli

```diff
@@ -0,0 +1,9 @@
+(************************************************************************)
```

Comment:
What's the point of this empty file?

Reply:
This enables the OCaml warning for unused code in configure.ml , otherwise, the compiler will assume that every top level definition is exported.